doc/manual.md

```diff
@@ -5160,7 +5160,7 @@ possibly raised exceptions; the algorithm operates on `p`'s call graph:
    raise `system.Exception` (the base type of the exception hierarchy) and
    thus any exception unless `T` has an explicit `raises` list.
    However, if the call is of the form `f(...)` where `f` is a parameter of
-   the currently analyzed routine it is ignored that is marked as `.effectsOf: f`.
+   the currently analyzed routine, it is ignored and is marked as `.effectsOf: f`.
```

Comment:
```suggestion
   the currently analyzed routine that is marked as `.effectsOf: f`, it is ignored.
```
Well, this seems to make more sense to me.

Reply:
Ah yes, that does make more sense.